Dockerfile.nginx

```diff
@@ -0,0 +1,2 @@
+FROM nginx:1.11.13-alpine
```

Comment:
Here's something to consider (this may be work for the 18F team to do): We have an nginx conf file that we use in prod in the repo. It handles things like passing requests to JSON and CSV files to the S3 bucket. Unfortunately, that file has the routes for those things hardcoded.

Our deployment environment allows us to use `erb` to replace those with env vars ([example](https://github.com/18F/federalist-proxy/blob/master/nginx.conf#L36)). If we could setup the Docker image here so it behaved similarly, that would be awesome, and would probably give folks using this to deploy to prod a better experience.